tests/counterpoll_test.py

```diff
@@ -76,6 +91,13 @@ def test_update_counter_config_db_status(self, status, _get_config_db_file):
             for counter, counter_config in config_db["FLEX_COUNTER_TABLE"].items():
                 assert counter_config["FLEX_COUNTER_STATUS"] == status
 
+    @pytest.mark.parametrize("status", ["disable", "enable"])
+    def test_update_pg_drop_status(self, status):
+        runner = CliRunner()
+        result = runner.invoke(counterpoll.cli.commands["pg-drop"].commands[status])
+        print(result.output)
+        assert result.exit_code == 0
```

Comment:
Please also validate that 'flex_counter_status' is accordingly updated in config db

Reply:
Done